spec/javascript/packages/document-capture/components/document-capture-warning-spec.jsx

```diff
@@ -148,17 +161,41 @@ describe('DocumentCaptureWarning', () => {
 
       it('renders with failed doc type', () => {
         const isFailedDocType = true;
-        const { getByRole, getByText } = renderCcontent(
+        const { getByRole, getByText, queryByText } = renderContent({
           isFailedDocType,
           isFailedResult,
           inPersonUrl,
-        );
+        });
 
         // error message section
         validateHeader('errors.doc_auth.doc_type_not_supported_heading', 1, true);
         validateHeader('errors.doc_auth.rate_limited_subheading', 2, false);
         expect(getByText(/general error/)).to.be.ok();
         expect(getByText(/idv.warning.attempts_html/)).to.be.ok();
+        expect(queryByText('idv.failure.attempts_html')).to.null();
+        expect(getByRole('button', { name: 'idv.failure.button.warning' })).to.be.ok();
+        // ipp section not existing
+        validateIppSection(false);
+        // troubleshooting section
+        validateTroubleShootingSection();
+      });
+
+      it('renders with successful selfie', () => {
```

Comment:
I think this description might be wrong
```suggestion
      it('renders with an unsuccessful selfie', () => {
```

Reply:
We may want to add an example for a successful selfie though, if there isn't one (I scanned and didn't see one, but I could have missed it).

Reply:
Never mind my last point - since this is the `document-capture-warning-spec` I think we wouldn't even be in this file unless there was some sort of warning to process (I still think the description on the `it` block needs to be fixed though).

Reply:
I looked at adding another example, but I think I agree: We shouldn't test that case, we don't expect (or allow) it to happen. Basically, we should expect that if we are showing the `document-capture-warning` then there must be something wrong.